_src/Anti-Debug/techniques/assembly.md

```diff
@@ -404,6 +405,48 @@ bool IsDebugged()
 
 <hr class="space">
 
+
+<br />
+<h3><a class="a-dummy" name="popf_and_cpuid">9. POPF and CPUID</a></h3>
+
+This technique is similar to <tt>7. POPF and Trap Flag</tt>.
```

Comment:
Please add the link to the technique, similar to the way it can be clicked from the contents of the page